remotespark/datawidgets/encoding.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2015  aggftw@gmail.com
+# Distributed under the terms of the Modified BSD License.
+
+
+class Encoding(object):
```

Comment:
What does the name "Encoding" mean? Even reading the rest of the class I'm unsure what its actual purpose is.  This class seems to hold chart settings, so maybe a name like "ChartSettings" would be more appropriate?